firebase-dynamic-links/src/main/java/com/google/firebase/dynamiclinks/PendingDynamicLinkData.java

```diff
@@ -100,6 +105,21 @@ public Uri getLink() {
     return null;
   }
 
+  /**
+   * Return the {@link Bundle} which contains utm parameters associated with the firebase dynamic
+   * link.
+   *
+   * @return Bundle of utm parameters associated with firebase dynamic link.
+   */
+  @NonNull
+  public Bundle getUtmParameters() {
```

Comment:
should this be `getUTMParameters` since UTM is an abbreviation

Reply:
Based on offline discussion, we decided to go with getUtmParameters itself